### ISSUE TITLE
[Accessibility] List items under 'Snooze for' dropdown list is not accessible through the swipe gesture.

### ISSUE THREAD

**Issue by RebeccaAnne** [microsoft/AdaptiveCards#6290]
### Target Platforms

iOS

### SDK Version

App Version: Version 1.0 (2.3.1-beta.20210603.1)

### Application Name

Visualizer

### Problem Description

[33755242](https://microsoft.visualstudio.com/OS/_workitems/edit/33755242)

List items under 'Snooze for' dropdown is not accessible through the the swipe gesture.

### Card JSON

[CalendarReminder.json](https://github.com/microsoft/AdaptiveCards/blob/main/samples/v1.0/Scenarios/CalendarReminder.json)

**Comment by jwoo-msft** [comment]
verified that it's fixed in the current build

**Comment by vagpt** [comment]
Working fine on the latest build hence closing this issue.
